src/utils/mod.rs

```diff
@@ -1062,7 +1084,7 @@ impl From<PcrSelections> for TPML_PCR_SELECTION {
         for (hash_algorithm, pcr_slots) in pcr_selections.items {
             let tpms_pcr_selection = &mut ret.pcrSelections[ret.count as usize];
             tpms_pcr_selection.hash = hash_algorithm.clone().into();
-            tpms_pcr_selection.sizeofSelect = pcr_selections.size_of_select;
+            tpms_pcr_selection.sizeofSelect = pcr_selections.size_of_select.to_u8().unwrap();
```

Comment:
Sorry to be annoying about `unwrap` again 😬 
We try to apply the following rule about panicking in our repos:
* remove all panicking behaviour if possible (`panic`, `expect`, `unwrap`, `unreachable`, etc) and return errors instead
* if in some location it is sure (proven) that a panic will not happen, we put a comment next to it saying why it is the case

We do it that way to make our libraries more stable as a whole, write them in what we think is a gode code quality practice but most importantly for security reasons: any panicking is a potential path opened to create a denial of service attack.

In that case, `enum PcrSelectSize` will probably not have a value outside of 0 - 255 so I'd say it is fine to panic, even better if there is a comment saying that 😋 

Don't worry about changing that now though! Just wanted to point out our "panicking guidelines".